SystemInfo/Collector/ComposerSystemInfoCollector.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * File containing the ComposerSystemInfoCollector class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\SystemInfo\Collector;
+
+use EzSystems\EzSupportToolsBundle\SystemInfo\Value;
+
+/**
+ * Collects information about installed Composer packages.
+ */
+class ComposerSystemInfoCollector implements SystemInfoCollector
```

Comment:
What about `JsonComposerSystemInfoCollector` ? We might build a different implementation that uses a lib of some sorts.